app/src/main/java/ru/vssemikoz/newsfeed/models/Category.java

```diff
@@ -15,24 +15,24 @@
 
     private Integer categoryId;
 
-    public static String getCategoryName(Category category){
+    public static String getCategoryName(Category category) {
         return category.toString();
     }
 
-    Category(int stringCategory){
+    Category(int stringCategory) {
```

Comment:
`stringCategory` means that you're using a string representation of the category. The type of this variable is `int` so it can't be a `string`. It's more about `categoryResId`.